src/PSRule.Rules.Azure/rules/Azure.ACR.Rule.ps1

```diff
@@ -48,4 +48,13 @@ Rule 'Azure.ACR.GeoReplica' -Ref 'AZR-000004' -Type 'Microsoft.ContainerRegistry
     return $Assert.Fail($LocalizedData.ReplicaNotFound);
 }
 
+# Synopsis: Azure Container Registries should have soft delete policy enabled and retention period to 90 days.
+Rule 'Azure.ACR.SoftDelete' -Ref 'AZR-000299' -Type 'Microsoft.ContainerRegistry/registries' -Tag @{ release = 'Preview'; ruleSet = '2022_09'; } {
```

Comment:
Use `AZR-000310` as there is several PRs and recently merged change that clash. Which is the reason for the build error.

```suggestion
Rule 'Azure.ACR.SoftDelete' -Ref 'AZR-000310' -Type 'Microsoft.ContainerRegistry/registries' -Tag @{ release = 'Preview'; ruleSet = '2022_09'; } {
```